### PR TITLE
WoV: Set up for translations

### PR DESCRIPTION
**Pull request by stevecotton**
The next scons or cmake pot-update will create all of the empty .po files.

This pushes the Portuguese translation below the 70% threshold.

For the en_GB translation, AFAICS only "honor" and maybe "overwatch" need to change.

### CODE REVIEW

File: po/wesnoth-wov/wesnoth-wov.pot [hunk 1]
```diff
@@ -0,0 +1,2109 @@
...
+#. [message]: speaker=Krenix
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:223
+msgid "Thatsss not a dragon."
+msgstr ""
+
+#. [message]: speaker=Xirtrezyx
+#: data/campaigns/Wings_of_Victory/scenarios/08_Wesmere.cfg:227
+msgid ""
+"Sssilence! I can sssee that now that we are up close, but the dragon has "
+"beaten the humans and the elvesss, so do not disrespect the dragon."
```
Comment (stevecotton) [inline po/wesnoth-wov/wesnoth-wov.pot]: @sigurdfdragon, should this be "humansss"?

BTW, I love this line. It was great in the campaign, and it's still great looking at it multiple times.

**Comment by stevecotton** [comment]
Checking the build failures, it seems that all the template .po files do need to be included.
@ivanovic should I include the en_GB with all strings marked as translated (and honor changed to honour)?

The Appveyor failure is unrelated, it's the removal of sample_user_handler.cpp. Will rebase with the .po files added when that's fixed.

**Comment by Wedge009** [comment]
70% being the threshold as to whether or not a translation can be selected?

Let me know when it's ready - I'll still review en_GB since there's more to localisation than just spelling differences.

**Comment by stevecotton** [comment]
It's now ready. All of the .po files are just templates, including the en_GB one.

> 70% being the threshold as to whether or not a translation can be selected?

I thought so, but was wrong. The threshold is 80% of core, without the campaigns, and the percentages in data/language are only for core without the campaigns.

File: po/wesnoth-wov/wesnoth-wov.pot (hunk 1 above)
Comment (sigurdfdragon) [reply po/wesnoth-wov/wesnoth-wov.pot]: possibly

**Comment by sigurdfdragon** [comment]
@stevecotton Thanks

**Pull request merged**